webapp/components/survey/SurveyForm/nodeDefs/nodeDefUIProps.js

```diff
@@ -140,6 +142,11 @@ export const getFormFields = getProp('formFields', ['field'])
 
 export const getDefaultValue = getProp('defaultValue')
 
+export const getValidationsEnabled = (nodeDefType) => {
+  const enabled = getPropByType('validations')(nodeDefType)
+  return !A.isNull(enabled) ? enabled : true
+}
```

Comment:
the name should be getValidationsEnabledByType and it can be simply 
```
export const getValidationsEnabled = getPropByType('validations', true)
```

Reply:
Changed!